src/components/ExternalLink.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useCallback } from "react";
+import { Icons, Link as BaseLink } from "@ledgerhq/native-ui";
+import { LinkProps } from "@ledgerhq/native-ui/components/cta/Link";
+import { track } from "../analytics";
+
+type Props = {
+  disabled?: boolean;
+  event?: string;
+  eventProperties?: Object;
+  Icon?: React.ComponentType<{ color: string; size: number }>;
+  /** deprecated, use `iconPosition` instead */
+  iconFirst?: boolean;
+  iconPosition?: LinkProps["iconPosition"];
+  onPress?: LinkProps["onPress"];
+  text: LinkProps["children"];
+  type?: LinkProps["type"];
+};
+
+export default function ExternalLink({
```

Comment:
> Reimplementation of LLM's `ExternalLink` component to use the UI lib's `Link` component. Here I **removed some props** regarding styling etc. (`ltextProps`, `color`, `style`, `fontSize`) so **the changes are potentially breaking but only style-wise**, and since we use the UI lib there shouldn't be anything visually "incorrect". We'll fix as we go I guess 🤷‍♂️